lib/Toadfarm/Plugin/Reload.pm

```diff
@@ -22,11 +22,17 @@ Go to "https://github.com/YOUR-USERNAME/YOUR-REPO/settings/hooks" to set it up.
 
 =item *
 
-The WebHook URL needs to be "http://yourserver.com/some/secret/path" and
+The WebHook URL needs to be "http://yourserver.com/some/private/path" and
 should not trigger any of the mounted apps, e.g. with
 L<virtual hosts|Toadfarm::Manual::VirtualHost>, the hostname part of the
-WebHook URL need to be unique from the mounted apps. One way to do this is
-simply to use an IP address instead of a hostname.
+WebHook URL needs to be unique from the mounted apps. This can be achieved by
```

Comment:
I think this sentence is contradicting. "needs to be" doesn't make any sense imo, when it can be avoided with your "hack".